frontend/.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 		es6: true,
 	},
```

Comment:
Reproduction steps:
1. Add step in creator
2. Change some edge to boundary
3. Get back to the previous step
4. Change the same edge to valley

```
Uncaught TypeError: edgesPerAssignment[assignment] is undefined
```